src/Social/Services/Messages.php

```diff
@@ -44,6 +45,34 @@ public static function getMessageByUuid(string $uuid): MessagesInterface
         ]);
     }
 
+    /**
+     * Get all the messages of a user.
+     *
+     * @param UserInterface $user
+     * @param integer $limit
+     * @param integer $page
+     * @return Simple
+     */
+    public static function getByUser(UserInterface $user, int $page = 1, int $limit = 25): Simple
+    {
+        $feed = new UserMessages();
+        return $feed->getUserFeeds($user, $limit, $page);
+    }
+
+    /**
+     * Get all the messages of a channel
+     *
+     * @param Channels $user
+     * @param array $filter
+     *
+     * @return Simple
+     */
+    public static function getByChannel(Channels $channel, int $page = 1, int $limit = 25, string $orderBy = "id", string $sort = "DESC", string $messageTypeId = null): Simple
```

Comment:
@rwhite27 shouldn it be ?string $messageTypeId = null

Reply:
that function is no longer necessary for this  version. Will tag it as deprecated.